client/src/components/lower-info-area/new-and-better-way.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-container id="new-and-better-way" class="pt-16">
+    <v-row no-gutters class="pa-9">
+      <v-col cols="6" class="pl-3 pr-75">
+        <v-img src="../../assets/images/bcnamerequest-laptop_x2.jpg" />
+      </v-col>
+      <v-col cols="6" class="pr-3 pl-75">
```

Comment:
What do these columns look like when the window is resized?

With your right and left padding in here, I wonder if a different layout or alignment system would work better?

Aligning content in a col: https://stackoverflow.com/questions/57769340/vuetify-right-align-group-of-buttons

Or flexbox: https://vuetifyjs.com/en/styles/flex/

PS If it's OK currently then don't change it, but the large padding makes me wonder.

Reply:
tracey mentioned about spacing required from the middle, min of 75px. It should be fine in big screen since we have a max-width of 1300px for the entire page (yet to implement)